contrib/auto-render/test/auto-render-spec.js

```diff
@@ -157,14 +169,14 @@ describe("A delimiter splitter", function() {
             ]);
 
         /* TODO(emily): make this work maybe?
-        expect("hello \\( ( world ) boo").toSplitInto(
-            "(", ")",
-            [
-                {type: "text", data: "hello \\( "},
-                {type: "math", data: " world ",
-                    rawData: "( world )", display: false},
-                {type: "text", data: " boo"},
-            ]);
+           expect("hello \\( ( world ) boo").toSplitInto(
+           "(", ")",
+           [
+           {type: "text", data: "hello \\( "},
+           {type: "math", data: " world ",
+           rawData: "( world )", display: false},
+           {type: "text", data: " boo"},
+           ]);
         */
```

Comment:
I think we should have a test like this with a mismatched delimiter, but it should probably throw.